src/internal/stats/stats.go

```diff
@@ -15,12 +15,6 @@ type ReadWrites struct {
 	ResourceOwners int   `json:"resourceOwners,omitempty"`
 }
 
-// Errs tracks the aggregation of errors that occurred during a process.
-type Errs struct {
```

Comment:
I'm all for removing dead code but are we sure this is safe to do? If this struct is getting serialized into the backup model then won't removing it break deserialization? It seems like either we'd need a custom deserializer or we'd need to keep the field around if this is deserialized

Reply:
Extra elements don't break serialization.  They do get sliced out.  But we're not actually utilizing this value for anything other than the error counts on backup list (if that's even serializing properly in the first place; i haven't checked specifically, but errors in general aren't serializing correctly).  Technically a breaking change, but a benign one.  I do need to add a Changelog for it, though.

Reply:
That seems... a bit scary. I also forgot about that behavior :/ Do we assume we'll never want to be more strict about unmarshalling (there's a way to have it error on unknown fields)?

Reply:
Error marshalling/unmarshalling is getting solved in #2708.

Reply:
ah, not just unmarshalling for errors. But for any type. Golangs json package has an option to throw errors if it encounters unknown fields